_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced `aria-disabled` behaviors to match `disabled` behaviors.
+    summaryAdditional: Now, the input element and calendar icon remain discoverable to screen readers but prevent the user from selecting or inputting a date.
```

Comment:
**suggestion:** Recommend we make this summary simpler to focus more on what the user will care about. 

```suggestion
    summary: Added `aria-disabled` to the list of expected attributes. 
    summaryAdditional: Now, the component will disable toggle when the `aria-disabled` attribute is present.
```

Reply:
Resolved in [a591638](https://github.com/uswds/uswds-site/pull/2794/commits/a59163890d9f3e277ca43f34a3dfd72c3d878a2b)